pkg/middleware/rbac.go

```diff
@@ -33,11 +38,12 @@ func NewRbac(config Rbac) echo.MiddlewareFunc {
 	if config.Client == nil {
 		panic("client cannot be nil")
 	}
+
 	return func(next echo.HandlerFunc) echo.HandlerFunc {
 		return func(c echo.Context) error {
 			logger := zerolog.Ctx(c.Request().Context())
 			path := MatchedRoute(c)
-			if config.Skipper != nil && config.Skipper(c) {
+			if config.Skipper != nil && config.Skipper(c) || utils.Contains(skipRbacRoutes, path) {
```

Comment:
Couple things here 
- If I do "/api/content-sources/**v1**/templates/:template_uuid/config.repo", this won't skip the endpoint.
- also, we don't want to skip rbac if a user is making the request, just a system

Reply:
And one idea
- I like the simplicity of your idea here, checking aginast the path returned by MatchedRoute. 
- We already have this method [SkipAuth](https://github.com/content-services/content-sources-backend/blob/18eb2c891412ac3fd50b4abdb5b7d43c6ba15853/pkg/middleware/enforce_identity.go#L36), which is similar (but is for skipping more than just rbac). So my first thought is maybe we can use your idea to simplify this function.
- I think it would make sense to have a SkipRbac function and then a function that wraps both of these functions when defining the Rbac middleware. This way we skip the middleware in a central place and in a similar for both use cases.